docs/source/membership/membership.md

```diff
@@ -1,341 +1,194 @@
+
+* **cacerts:** This folder contains a list of self-signed X.509 certificates of the Root CAs trusted by the organization represented by this MSP. There must be at least one Root CA certificate in this MSP folder.
+
+  This is the most important folder because it identifies the CAs from which all other certificates must be derived to be considered members of the
+  corresponding organization to form the chain of trust.
+
+* **intermediatecerts:** This folder contains a list of X.509 certificates of the Intermediate CAs trusted by this organization. Each certificate must be signed by one of the Root CAs in the MSP or by any Intermediate CA whose issuing CA chain ultimately leads back to a trusted Root CA.
+
+  An intermediate CA may represent a different subdivision of the organization (like `ORG1-MANUFACTURING` and `ORG1-DISTRIBUTION` do for `ORG1`), or the
+  organization itself (as may be the case if a commercial CA is leveraged for the organization's identity management). In the latter case intermediate CAs
+  can be used to represent organization subdivisions. [Here](../msp.html) you may find more information on best practices for MSP configuration. Notice, that
+  it is possible to have a functioning network that does not have an Intermediate CA, in which case this folder would be empty.
+
+  Like the Root CA folder, this folder defines the CAs from which certificates must be issued to be considered members of the organization.
+
+* **admincerts (Deprecated from Fabric v1.4.3 and higher):** This folder contains a list of identities that define the actors who have the role of administrators for this organization. In general, there should be one or more X.509 certificates in this list.
+
+  **Note:** Prior to Fabric v1.4.3, admins were defined by explicitly putting certs in the `admincerts` folder in the local MSP directory of your peer. **With Fabric v1.4.3 or higher, certificates in this folder are no longer required.** Instead, it is recommended that when the user is registered with the CA, that the `admin` role is used to designate the node administrator. Then, the identity is recognized as an `admin` by the Node OU role value in their signcert. As a reminder, in order to leverage the admin role, the "identity classification" feature must be enabled in the config.yaml above by setting "Node OUs" to `Enable: true`. We'll explore this more later.
+
+  It's worth noting for Channel MSPS that just because an actor has the role of an administrator it doesn't mean that they can administer particular resources. The actual power a given identity has with respect to administering the system is determined by the _policies_ that manage system resources. For example, a channel policy might specify that `ORG1-MANUFACTURING` administrators have the rights to add new organizations to the channel, whereas the `ORG1-DISTRIBUTION` administrators have no such rights.
+
```

Comment:
Channel MSPs. The point made by this paragraph is something that should be mentioned earlier in the section relating to the Channel MSP. I just made it in one of my earlier comments. Perhaps that language could be combined with this language.

Reply:
Combined with your earlier comment I added this information to the Organization and Node OUs section. I think that is the best place as it ties it all together there.